CompilerSource/ipc/Background.fbs

```diff
@@ -13,8 +13,8 @@ table Background {
   tile_height:int;
   horizontal_offset:int;
   vertical_offset:int;
-  horizontal_separation:int;
-  vertical_separation:int;
+  horizontal_spacing:int;
+  vertical_spacing:int;

```

Comment:
I don't mean to nitpick, but I just like the field name being a little shorter. I think spacing is a lot clearer too, in addition to being more common terminology used in GUI frameworks.